runtime/protocol/http-client-engines/http-client-engine-crt/jvm/test/aws/smithy/kotlin/runtime/http/engine/crt/RequestUtilTest.kt

```diff
@@ -11,36 +11,35 @@ import kotlin.test.assertTrue
 class RequestUtilTest {
     @Test
     fun testExceptionRetryability() {
-        setOf(
+        mapOf(
             // All IO errors are retryable
-            "AWS_IO_SOCKET_NETWORK_DOWN",
-            "AWS_ERROR_IO_OPERATION_CANCELLED",
-            "AWS_IO_DNS_NO_ADDRESS_FOR_HOST",
+            1050 to "AWS_IO_SOCKET_NETWORK_DOWN",
```

Comment:
**question**: How stable are these error codes? The way they are defined and used in C means you generally are never going to rely on specific numbers. I don't know if they guarantee they'll never insert one in-between existing ones.